src/mlpack/tests/activation_functions_test.cpp

```diff
@@ -720,9 +720,9 @@ BOOST_AUTO_TEST_CASE(HardSigmoidFunctionTest)
                                          0.2");
 
   CheckActivationCorrect<HardSigmoidFunction>(activationData,
-      desiredActivations);
```

Comment:
Hey I am fine with this either way but if you referred to style guide lines we can also use two tabs (each of two space to indent as well).

Reply:
yeah @kartikdutt18, they both are correct. I find this requires no eye movement🤭.